promise/tests/test-promise.js

```diff
@@ -131,7 +131,7 @@ o.spec("promise", function() {
 				callAsync(function() {resolve(promise)})
 			})
 
-			promise.then(null, done)
+			promise.then(null, () => done())
```

Comment:
Yes, IMO tests are fine to use arrows but library has to use ES5 funcs for compat w/o transpilation.

This sounds like a good danger rule... 😄